code_src/staking/dot/nominator/nominatorArgParser.py

```diff
@@ -0,0 +1,86 @@
+import argcomplete
+from common import MyHelpFormatter
+
+from code_src.staking.dot.dotArgparserUtil import actionSeed, actionValidatorAddress, actionHelp
+
+
+def nominateArgParser(nominatorSubParser):
+    example_bondextra = """
+description : 
+  nominate dot coin
+
+example:
+  python %(prog)s -s/--seed "MNEMONIC_PHRASE" -va/--validator_address VALIDATOR_ADDRESS \n
+  python %(prog)s -s/--seed "MNEMONIC_PHRASE" -va/--validator_address VALIDATOR_ADDRESS_1,VALIDATOR_ADDRESS_2,VALIDATOR_ADDRESS_N \n
+    """
+    nominateParser = nominatorSubParser.add_parser("nominate", help="""
+Declare the desire to nominate `targets` for the origin controller. Effects will be felt at the beginning of the next era.""",
+                                                   add_help=False, epilog=example_bondextra,
+                                                   formatter_class=MyHelpFormatter)
+    nominateParser.set_defaults(func="nominate")
+    nominateParser._action_groups.pop()
+
+    # bound extra group
+    nominateRequiredArguments = nominateParser.add_argument_group('required arguments')
+    nominateOptionalArguments = nominateParser.add_argument_group('optional arguments')
+
+    actionSeed(nominateRequiredArguments)
+    actionValidatorAddress(nominateOptionalArguments)
+    actionHelp(nominateOptionalArguments)
+
+
+# TODO check the right wat to unnominate
```

Comment:
Can you explain this TODO

Reply:
refer to this https://wiki.polkadot.network/docs/maintain-guides-how-to-unbond#step-2-unbonding-an-amount
i want to make sure that coin doesn't stuck or something is happen like i want to develop the right approach.